### PR TITLE
Abstracts mirror data attributes into a Mirrored component

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: Components/Mirrored.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { Component, PropTypes } from 'react';
```
Comment (joaojusto) [inline Components/Mirrored.js]: You don't need to import `Component`

**Review by joaojusto — APPROVED** [review]
Check my comment and :shipit:

File: Components/Mirrored.js (hunk 1 above)
Comment (naps62) [reply Components/Mirrored.js]: fixed

**Comment by joaojusto** [comment]
@naps62 :shipit:

**Pull request merged**